lib/fastlane/actions/pubu.rb

```diff
@@ -0,0 +1,80 @@
+module Fastlane
+  module Actions
+    # To share this integration with the other fastlane users:
+    # - Fork https://github.com/KrauseFx/fastlane
+    # - Clone the forked repository
+    # - Move this integration into lib/fastlane/actions
+    # - Commit, push and submit the pull request
+
+    # noinspection ALL
+    class PubuAction < Action
+      def self.run(params)
+        url = ENV['PUBU_URL']
+        message = params[:message].to_s
+        pubuayload = build_payload(message, params)
+        send_pubu(url, pubuayload)
+        raise "`PUBU_URL` lost  please set ENV first`".red unless url
+      end
+
+      def self.available_options
+        # Define all options your action supports.
+
+        # Below a few examples
+        [
+            FastlaneCore::ConfigItem.new(key: :message,
+                                         description: "The message will displayed on PubuIM",
+                                         optional: true),
+
+            FastlaneCore::ConfigItem.new(key: :pubuUrl,
+                                         env_name: "PUBU_URL",
+                                         description: "PubuIM services url", #todo add doc url here
```

Comment:
Annotation keywords like `todo` should be all upper case, followed by a colon, and a space, then a note describing the problem.<br>Missing space after #.